### DISCUSSION TITLE
Layer Diffusion

### DISCUSSION THREAD

**Discussion by logikstate** [Acly/krita-ai-diffusion#462] [Ideas]
This looks interesting

https://github.com/layerdiffusion/sd-forge-layerdiffuse

**Comment by motolo** [comment]
I absolutely agree with you and among other things there is also the project for comfyui so I think it can be implemented with KritaAi

 https://github.com/huchenlei/ComfyUI-layerdiffuse

 Krita has great layer management and this could be a game changer :)

**Comment by guiteubeuh** [comment]
+1 I totally agree this would be fantastic.
Oh and by the way, kudos for your amazing work to the open source community Acly !

**Comment by canochento** [comment]
totally on the same board

down there an image with a workflow as an example.
![ComfyUI_temp_thnzp_00002_](https://github.com/user-attachments/assets/444bc652-7acd-4ebe-baf3-a637fc8d3856)

**Comment by paprik** [comment]
Started working on this in my fork - only works with Generate currently: https://github.com/paprik/krita-ai-diffusion/tree/layer-diffusion

Have an PR for it: https://github.com/Acly/krita-ai-diffusion/pull/1150
I'll try to keep you updated.
